requirements.txt

```diff
@@ -3,3 +3,5 @@ grpcio>=1.60.0 # 1.60.0 is the version introducing protobuf 1.25.X support, newe
 protobuf
 pytest
 pytest-cov
+azure-core
```

Comment:
Doesn't `azure-identity` import `azure-core` anyway? Maybe remove this unless you intent to set a minimum version.

Please see also my comment on the `pyproject` to specify a minimum version for `azure-identity`